src/containers/CodeReminderModal/CodeReminderModal.test.jsx

```diff
@@ -170,7 +170,7 @@ describe('CodeReminderModalWrapper', () => {
 
     expect(wrapper.find('.bulk-selected-codes').text()).toEqual('Selected codes: 2');
     expect(wrapper.find('#email-template')).toBeTruthy();
-    wrapper.find('.modal-footer .code-remind-save-btn').hostNodes().simulate('click');
+    wrapper.find('.pgn__modal-footer .code-remind-save-btn').hostNodes().simulate('click');
```

Comment:
nit: might recommend not relying on `.pgn` class names from Paragon, if possible, as class names are technically considered implementation details that theoretically could change upstream.